apps/dtlaunch/app-b2.js

```diff
@@ -125,12 +132,47 @@ Bangle.on("touch",(_,p)=>{
         }
     }
     if ((i==4 || (page*4+i)>Napps) && selected>=0) {
-        draw_icon(page,selected,false);
+        drawIcon(page,selected,false);
         selected=-1;
     }
-});
+};
+Bangle.on("touch",touchListenerDt);
+  
+const returnToClock = function() {
+  Bangle.setUI();
+  delete s;
+  delete a;
+  delete n;
+  delete Napps;
+  delete Npages;
+  delete maxPage;
+  delete selected;
+  delete oldselected;
+  delete page;
+  delete XOFF;
+  delete YOFF;
+  delete x;
+  delete y;
+  delete txt;
+  delete lineY;
+  delete line;
+  delete c;
+  delete O;
+  delete x1;
+  delete x2;
+  delete i;
+  delete drawIcon;
+  delete drawPage;
+  delete isTouched;
```

Comment:
Most of these are not needed, everything defined in the block scope with let/var can be garbage collected automatically. It can be useful to delete some big vars beforehand to have more memory for the app that should be loaded.

Reply:
And the garbage collector should be as fast having delete statements like this, or probably faster?

Reply:
Removed a bunch of delete statements. Unsure if those left should be removed as well.

Reply:
Yes, the GC will be faster - when the block goes out of scope, everything will be freed in one go